applicationinsights.dev.json

```diff
@@ -28,6 +28,22 @@
             }
           ],
           "percentage": 10
+        },
+        {
```

Comment:
Do you know why these are here - is it because they are not seen as dependencies by default?

Reply:
they flood app insights these so need to be ignored